pelican/tools/templates/fabfile.py.in

```diff
@@ -3,7 +3,10 @@ import fabric.contrib.project as project
 import os
 import shutil
 import sys
-import SocketServer
+if sys.version_info[0] >= 3:
```

Comment:
Even though it might be preferred in the docs (note however, they propose the old version under `try`), the motivation seems to be not treating the future as an exception and to ensure future compatibility. But I think that the code as it stands already lives up to that, even more so than the "official" preference because that goes for the old version first. Further I think my solution more clearly communicates that this is something where we expect different behaviour based on version.

I can still change it of course, if it is necessary.